images/contact.seq

```diff
@@ -0,0 +1,24 @@
+# This is a sequence diagram for the dkim section
+# I used https://bramp.github.io/js-sequence-diagrams/
+# to render it to svg and then save it in images folder
+
+participant Alice as Ua
+participant Bob as Ub
+
+Ua --> Ub: 1.a) bootstrap code
+Note over Ua: 1.b) track bootstrap per INVITENUMBER
+Note over Ub: 2.a) check for exising key
+Ub ->> Ua: 2.b) vc-request message with INVITENUMBER
+Note over Ua: 3.a) look up bootstrap by INVITENUMBER
+Note over Ua: 3.b) abort if invite expired
+Note over Ua: 3.c) process AC header
+Ua -> Ub: 3.d) vc-auth-required message with AC header
+Note over Ub: 4.a) abort if key does not match FP from bootstrap
+Ub -> Ua: 4.b) vc-request-with-auth with Bob_FP and AUTH
+Note over Ua: 5.a) verify AUTH and key
+Note over Ua: 5.b) on failure alert user and abort
+Note over Ua: 6.a) signal success to user
+Ua -> Ub: 6.b) vc-contact-confirm message
+Note over Ua: 6.c) clear bootstrap data for INVITECODE
+Note over Ub: 7. signal success to user
```

Comment:
i suggest using capital "A" and "B" -- it's more visually distiguisahble than "Ub" and "Ua" .

Reply:
Applied it directly to master without an additional PR.